templates/terraform/modules/environment/main.tf

```diff
@@ -162,3 +162,15 @@ module "user_access" {
   roles = var.roles
   users = local.users
 }
+
+module "cloudtrail" {
+  source = "commitdev/zero/aws//modules//cloudtrail"
```

Comment:
The double slash should only be at the root of the project, like this: `commitdev/zero/aws//modules/cloudtrail`
Mind changing `user_access` too?